exist-core/src/main/java/org/exist/storage/serializers/EXistOutputKeys.java

```diff
@@ -104,4 +104,14 @@ public class EXistOutputKeys {
      * Set to "yes" to enable xdm-serialization rules, false otherwise.
      */
     public final static String XDM_SERIALIZATION = "xdm-serialization";
+
+    /**
+     * Enforce newline at the end of an XML document.
+     *
+     * Since a lot of editors set this enforcing it on
+     * serialization out of exist-db will lead to less
+     * meaningless changes in git and tools like diff will
+     * be able to provide more meaningful as well.
+     */
+    public final static String INSERT_FINAL_NEWLINE = "insert-final-newline";
```

Comment:
`public static final` - otherwise you will get static code complaints.

Reply:
as discussed in the community call this should be done in a separate PR

Reply:
> as discussed in the community call this should be done in a separate PR

That is not my recollection of the discussion at all. My memory is that as this is new code it should be corrected.
You yourself even pointed out that you understood that others were opposed to fixing such issues in other large refactoring PRs.

Reply:
This is a one line addition to existing code where it follows the structure of the existing declarations. @dizzzz specifically said in the call that if you change one of the lines all of them should be changed, which in turn would obscure the PRs changes.
I am wondering if the order of keywords is blocking this PR.